docs/reference/deploy/helm-config-reference.rst

```diff
@@ -90,11 +90,10 @@
       nodes used by Determined are running in a service account that has the "Storage Object Admin"
       role on the GCS bucket being used for checkpoints. As an alternative (or when running outside
       of GKE), you can add the appropriate `service account credentials
-      <https://cloud.google.com/docs/authentication/production#obtaining_and_providing_service_account_credentials_manually>`__
-      to your container (e.g., via a bind-mount), and then set the
-      ``GOOGLE_APPLICATION_CREDENTIALS`` environment variable to the container path where the
-      credentials are located. See :ref:`environment-variables` for more information on how to set
-      environment variables in trial environments.
+      <https://cloud.google.com/iam/docs/service-account-creds>`__ to your container (e.g., via a
```

Comment:
I think this is the relevant tutorial: `https://cloud.google.com/docs/authentication/provide-credentials-adc#attached-sa`